yt_dlp/extractor/zingmp3.py

```diff
@@ -380,20 +447,29 @@ def _fetch_page(self, user_id, url_type, page):
         })
 
     def _real_extract(self, url):
-        user_alias, url_type = self._match_valid_url(url).group('user', 'type')
+        alias, url_type = self._match_valid_url(url).group('user', 'type')
         if not url_type:
             url_type = 'bai-hat'
 
-        user_info = self._call_api('info-artist', {}, user_alias, query={'alias': user_alias})
-        if url_type in ('bai-hat', 'video'):
-            entries = self._paged_list(user_info['id'], url_type)
+        user_info = self._call_api('info-artist', {}, alias, query={'alias': alias})
+
+        # Handle for new-release
+        if alias == 'new-release' and url_type in ('song', 'album'):
+            self._IE_NAME = 'zingmp3:NewRelease'
```

Comment:
as discussed in a previous comment, we should not change the `IE_NAME` during extraction
```suggestion
```